src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,16 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    // Prefix associated with Person
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
```

Comment:
```suggestion
    public static final Prefix PREFIX_TAG = new Prefix("t/");
    public static final Prefix PREFIX_GENDER = new Prefix("g/");
    
```

Hopefully this helps resolve the merge conflict? I don't think it will pose any issue.

Reply:
Can add "d/" for PREFIX_DOB as well but as said not big issue

Reply:
done